### ISSUE TITLE
not returning dictionary of the results

### ISSUE THREAD

**Issue by BrianPainter** [nerium-data/sqla-raw#21]
when I run db.result(<<my sql statement>>) I get:
`TypeError: cannot convert dictionary update sequence element #0 to a sequence`

I don't see where the code is getting the column names and adding those to the tuple or dictionary. the only thing that I can successfully return is the 'proxy'

**Comment by tym-xqo** [comment]
Hey @BrianPainter Sorry I didn't see that you'd put in this issue in until just recently. I have been away from the project for a bit, but actively working a new version now. I am aware of the issue; it is due to changes in SQLAlchemy 2.x, which changed the interface to `sqlalchemy.engine.row.Row`. I plan to have a new release of `sqla-raw` out in the next few days that will pin that at < 2, and follow that up with another that will support latest SQLA version.

In the meantime, you can work around the issue by installing an earlier SQLA yourself before `sqla-raw`:

```
$ python -m venv .venv
$ source .venv/bin/activate
$ python -m pip install "sqlalchemy<2"
$ python -m pip install sqla-raw
```

Hope that helps!

**Comment by tym-xqo** [comment]
Fixed by #24

**Comment by tym-xqo** [comment]
Fixed by https://github.com/nerium-data/sqla-raw/pull/24
